lms/djangoapps/teams/views.py

```diff
@@ -58,6 +60,9 @@
 MAXIMUM_SEARCH_SIZE = 100000
 
 
+log = logging.getLogger(__name__)
+
+
 class TeamsDashboardView(View):
```

Comment:
Nit: you didn't name this, but I'm not sure why this is called a dashboard view. We don't have the terminology of a dashboard anywhere...